src/applications/vaos/actions/appointments.js

```diff
@@ -70,21 +70,24 @@ export function fetchFutureAppointments() {
         const data = await Promise.all([
           getConfirmedAppointments(
             'va',
-            moment().format('YYYY-MM-DD'),
             moment()
-              .add(4, 'months')
-              .format('YYYY-MM-DD'),
+              .startOf('day')
+              .toISOString(),
+            moment()
+              .startOf('day')
+              .add(120, 'days')
+              .toISOString(),
           ),
           getConfirmedAppointments(
             'cc',
             moment().format('YYYY-MM-DD'),
             moment()
-              .add(4, 'months')
+              .add(120, 'days')
               .format('YYYY-MM-DD'),
           ),
           getPendingAppointments(
             moment()
-              .subtract(4, 'months')
+              .subtract(30, 'days')
```

Comment:
This is what I wanted to do initially, changed it because of the backend bug we figured out yesterday.